src/cli/nexus-prisma.ts

```diff
@@ -26,7 +25,15 @@ generatorHandler({
   },
   // async required by interface
   // eslint-disable-next-line
-  async onGenerate({ dmmf }) {
+  async onGenerate({ dmmf, otherGenerators }) {
+    const prismaClientGenerator = otherGenerators.find((g) => g.provider.value === 'prisma-client-js')
+
+    if (!Gentime.settings.data.prismaClientLocation) {
+      Gentime.changeSettings({
+        prismaClientLocation: prismaClientGenerator?.output?.value || '@prisma/client',
```

Comment:
```suggestion
        prismaClientLocation: prismaClientGenerator.output.value
```

Given the above change.